concepts/vocabulary.md

```diff
@@ -0,0 +1,31 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Vocabulary
+description: The set of supported words
+---
+
+The **vocabulary** of a model is the set of supported words.
+In a machine translation system, the vocabulary may refer to the set of words supported in either the input or the output.
+A word that's supported in the output is a word that can be generated by the translation model.
```

Comment:
According to our [Style guide](https://machinetranslate.org/style#apostrophes-and-quotation-marks), we should use the curly apostrophe. However, I think we can eliminate the `that's` altogether.

Reply:
Easier to eliminate it yeah.

I'm surprised that the style guide is meant to apply to word-internal quotes though, especially since the site is meant to be sent through MT. Many NLP systems don't handle "smart quotes" very well because it makes it a different token altogether unless it's merged in preprocessing.

Reply:
You are absolutely right, I must fix that.